File: testsuite/matlab_and_octave/util/fix_normalization_inconsistency.m
function y = fix_normalization_inconsistency (normalization)
	if (normalization==1)
		y='SQRT';
	elseif (normalization==2)
		y='FULL';
	elseif (normalization==3)
		y='SQRTLEN';
	elseif (normalization==4)
		y='LEN';
	elseif (normalization==5)
		y='SQLEN';
	else
		y='NO';
	end

File: testsuite/r/util/fix_normalization_inconsistency.R
fix_normalization_inconsistency <- function(normalization) {
	if (normalization==1) {
		return('SQRT')
	} else if (normalization==2) {
		return('FULL')
	} else if (normalization==3) {
		return('SQRTLEN')
	} else if (normalization==4) {
		return('LEN')
	} else if (normalization==5) {
		return('SQLEN')
	} else {
		return('NO')
	}
}
